src/node/modules/DiffieHellman.ts

```diff
@@ -0,0 +1,48 @@
+import {createECDH, ECDH} from 'crypto';
+
+export class DiffieHellman
+{
+    private readonly _prime256v1_uncompressed_curved_name_pem_header_hex = '3059301306072a8648ce3d020106082a8648ce3d030107034200';
+    private _public_key_pem?: string;
+    private _diffie_hellman: ECDH;
+
+    public constructor() {
```

Comment:
Please keep code style consistent and put this brace on a new line as the rest of the code